db/migrate/20220811093302_update_audits_generated_by_api_user.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+# rubocop:disable Rails/ReversibleMigration
+class UpdateAuditsGeneratedByApiUser < ActiveRecord::Migration[6.1]
+  # rubocop:disable Metrics/MethodLength
+  def change
+    execute(
+      "UPDATE audits
+      SET api_user_id = 1
+      WHERE api_user_id IS NULL
+      AND user_id IS NULL
+      AND activity_type IN (
+        'description_change_validation_request_received',
+        'replacement_document_validation_request_received',
+        'additional_document_validation_request_received',
+        'red_line_boundary_change_validation_request_received',
+        'other_change_validation_request_received'
+      );"
+    )
+
+    execute(
```

Comment:
Updates the 'document uploaded' audit associated with the additional document 'additional_document_validation_request_received' event.